src/test/java/com/palantir/ri/ResourceIdentifierTest.java

```diff
@@ -1,17 +1,5 @@
 /*
  * (c) Copyright 2015 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
```

Comment:
Was this intentional?

Reply:
Hmm, it's what spotless wants, but I don't know why: https://app.circleci.com/pipelines/github/palantir/resource-identifier/68/workflows/12becf1a-03f4-4461-95f1-4add110841d4/jobs/533

Reply:
Hmm, I see in the excavator it runs `rm -f .baseline/copyright/999_palantir.txt` one upgrading OSS things - did that now.